.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,89 @@
+name: conductor system tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 17 * * *'
+
+  push:
+    branches:
+      - main
+
+jobs:
+  system-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.11"]
+        poetry-version: ["1.4.2"]
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Run image
+        uses: abatilo/actions-poetry@v2
+        with:
+          poetry-version: ${{ matrix.poetry-version }}
+      - name: Clone ga_workflows_collected_data
+        if: ${{ ( success() || failure() ) &&  ( steps.execute_tests.outcome != 'skipped' ) }}
+        uses: actions/checkout@v3
+        with:
+          repository: FRINXio/ga_workflows_collected_data
+          ref: main
+          token: ${{ secrets.token_ga_workflows_collected_data }}
+          path: ./ga_workflows_collected_data
+      - name: Install package dependencies
+        run: poetry install
+      - name: Execute worker and dependency
+        run: docker compose -f tests/execute_sdk/docker-compose.yml up -d
+      - name: Run system tests
+        run: |
+          mkdir -p logs
+          start=`date +%s`
+          poetry run pytest tests/system_tests/ -vv \
+            --collect-stats-folder="ga_workflows_collected_data/" \
+            --junit-xml=logs/test-results.xml
+          end=`date +%s`
+          echo "DURATION=$((end-start))" >> "$GITHUB_ENV"
+      - name: Execute tests - measure duration
+        if: ${{ ( success() || failure() ) &&  ( steps.execute_tests.outcome != 'skipped' ) }}
+        run: |
+          FRINX_CONDUCTOR_VERSION=$(docker container inspect $(docker ps --filter name=conductor-server -q) --format '{{ .Config.Labels.git_commit }}')
+          FRINX_CONDUCTOR_WORKERS_AND_PYTHON_SDK_VERSION=$(git describe --tags)
+          filename=$(echo ${{ github.workflow }} | sed 's| |_|g')
```

Comment:
nice trick
I stubornly insist on not to have spaces in the name of workflow in our uniconfig repo
to avoid of problems such as this could cause

Reply:
Not sure what you mean. Can you explain it more?
$(echo ${{ github.workflow }} | sed 's| |_|g') should convert workflow name to conductor_system_tests